config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -14,6 +14,10 @@ presubmits:
     always_run: true
     path_alias: k8s.io/kubernetes
     spec:
+      # unit tests have no business requiring root or doing privileged operations
+      securityContext:
+        runAsUser: 2000
+        allowPrivilegeEscalation: false
```

Comment:
Should we drop all capabilities and also add `runAsGroup: 2000`?

Reply:
probably the right call, we should start in the -experimental job first though and make sure there isn't something depending on this (and if so fix it before rolling it out to everyone's PRs)

Reply:
closing the loop https://github.com/kubernetes/test-infra/pull/22780